src/NHSD.BuyingCatalogue.Ordering.Contracts/NHSD.BuyingCatalogue.Ordering.Contracts.csproj

```diff
@@ -0,0 +1,15 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net5.0</TargetFramework>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+    <DocumentationFile>D:\Project\GPITFutures\API\13236_ORDAPI\BuyingCatalogueOrdering\NHSD.BuyingCatalogue.Ordering.Contracts\NHSD.BuyingCatalogue.Ordering.Contracts.xml</DocumentationFile>
```

Comment:
Can you replace this element with the below element?

```xml
<GenerateDocumentationFile>true</GenerateDocumentationFile>
```

Reply:
Added `GenerateDocumentationFile` element